package.json

```diff
@@ -18,13 +18,10 @@
     "test:type": "npx lerna run test --scope \"@sap-cloud-sdk/type-tests\"",
     "generate:test-services": "npx ts-node generate-test-services.ts && npm run lint:fix && lerna link --force-local",
     "compile": "npx lerna run compile",
-    "lint": "npm run tslint && npm run prettier",
-    "lint:fix": "npm run tslint:fix && npm run prettier:fix",
+    "lint": "npx eslint --ext .ts .",
+    "lint:fix": "npm run lint -- --fix",
+    "lint:fix-cached": "npm run lint:fix -- --cache",
     "postinstall": "npx lerna bootstrap --hoist --force-local --strict",
-    "prettier": "prettier -l \"**/*.ts\"",
```

Comment:
why `prettier` is also removed?
if this is intended, then it should also be removed form the `devDependencies`

Reply:
This is intended, but we need the dev dependency. Eslint is now taking over the check, so we don't need any extra command for that.